docs/api/hardhat/plugins.md

```diff
@@ -411,3 +411,108 @@ expect(await contract.getAmount()).to.equal(100);
 ```
 
 Checkout the advantages of using chai matchers [here](https://hardhat.org/hardhat-chai-matchers/docs/overview#why-would-i-want-to-use-it?). Since the list of all supported chai matchers is same as with [hardhat-chai-matchers](https://hardhat.org/hardhat-chai-matchers/docs/overview) plugin, check the [reference documentation](https://hardhat.org/hardhat-chai-matchers/docs/reference).
+
+
+## `hardhat-zksync-verify`
+
+This plugin is used to verify contracts on the zkSync 2.0 network.
+
+### Npm
+
+[@matterlabs/hardhat-zksync-verify](https://www.npmjs.com/package/@matterlabs/hardhat-zksync-verify)
+
+This plugin is used in conjunction with [@nomiclabs/hardhat-etherscan](https://www.npmjs.com/package/@nomiclabs/hardhat-etherscan) and it supports backward compatibility with that plugin.
```

Comment:
Shall we add this as a warning to make sure users also install the @nomiclabs/hardhat-etherscan package? I'd also add this package in the installation script

Reply:
Sure, we can add the additional warning for users to install nomiclabs etherscan package. I will also include it in the installation script.